classes/03class/exercises/c03-docker06/carloshz4/README.md

```diff
@@ -0,0 +1,30 @@
+# C03-Docker06
+
+## Docker 
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+- Commands used to build  the image:
+```
+docker build -t hello-name:v1 .
+
+Sending build context to Docker daemon  2.048kB
+Step 1/2 : FROM alpine:latest
+ ---> a24bb4013296
+Step 2/2 : CMD echo "Hello $var"
+ ---> Running in e729e7ef5cbe
+Removing intermediate container e729e7ef5cbe
+ ---> 27c02b0a553f
+Successfully built 27c02b0a553f
+Successfully tagged hello-name:v1
+```
+
+- Commands used to execute the image and the output of container execution:
+```
+docker run --env var=Carlos hello-name:v1
```

Comment:
you should only run `docker run hello-name Carlos` to get the `Hello Carlos` response.

Reply:
Sure, thanks @kikobr82, I'll check how to do that.